### PR TITLE
[DOCS] #4883 – Borg documentation downplays nonce reuse issue

### PR DESCRIPTION
**Pull request by fantasya-pbem**
Improve the consistency of multi-client issues with same repository.
Add a comment in FAQ that it can improved further when #3428 is solved.

### CODE REVIEW

File: docs/internals/security.rst [hunk 1]
```diff
@@ -200,10 +201,11 @@ between clients, which is handled through the repository. Therefore in
...
 the manifest (which will fail if the client has seen a more recent
-manifest or has a more recent nonce reservation). If the repository is
-untrusted, but a trusted synchronization channel exists between
-clients, the security database could be synchronized between them over
-said trusted channel. This is not part of Borg's functionality.
+manifest or has a more recent nonce reservation).
+
+If the repository is untrusted, there *must* exists trusted channel
+between clients and security databases *must* be synchronized with the
+outlined protocol. This is not part of Borg's functionality.
```
Comment (ThomasWaldmann) [inline docs/internals/security.rst]: Well, if we don't have it, it doesn't help to talk about that it must exist.

File: docs/quickstart.rst [hunk 2]
```diff
@@ -400,6 +399,11 @@ For automated backups the passphrase can be specified using the
     A backup inside of the backup that is encrypted with that key/passphrase
     won't help you with that, of course.
 
+    In case you lose your repository, but have an older copy of it to restore
+    from, don't use that later for creating new backups – you would run into
+    security issues (reuse of nonce counter values). It is better to initialize
+    a new Borg repository.
```
Comment (ThomasWaldmann) [inline docs/quickstart.rst]: Guess this is only true if you also lost the clientside counter value at the same time.

File: docs/internals/security.rst (hunk 1 above)
Comment (fantasya-pbem) [reply docs/internals/security.rst:208]: I just took the comments in the issue and placed them here. If you think we should not change this phrase, I'll revert it.

File: docs/quickstart.rst (hunk 2 above)
Comment (fantasya-pbem) [reply docs/quickstart.rst]: Added "and security info" to "lose your repository".

File: docs/faq.rst [hunk 3]
```diff
@@ -27,7 +27,12 @@ which is slower.
 Can I backup from multiple servers into a single repository?
 ------------------------------------------------------------
 
-Yes, but in order for the deduplication used by Borg to work, it
+Yes, this is *possible* from the technical standpoint, but it is
+*not recommended* from the security perspective. BorgBackup is
+built upon a defined :ref:`attack_model` that cannot provide its
+guarantees for multiple clients using the same repository.
+
+In order for the deduplication used by Borg to work, it
```
Comment (ThomasWaldmann) [inline docs/faq.rst:35]: Also, ...

(because this is another point, not related to the security issue in previous paragraph)

File: docs/faq.rst [hunk 4]
```diff
@@ -89,6 +94,10 @@ There is also a similar encryption security issue for the disaster case:
 If you lose repo and the borg client-side config/cache and you restore
 the repo from an older copy-of-repo, you also run into AES counter reuse.
 
+.. In issue #3428 and PR #5034 we will introduce another FAQ titled
+.. "My repository is corrupt, how can I restore from an older copy of it?"
+.. When this is merged, we can link this FAQ to that, replacing the last paragraph.
+
```
Comment (ThomasWaldmann) [inline docs/faq.rst:100]: Merged now.

File: docs/internals/security.rst (hunk 1 above)
Comment (ThomasWaldmann) [reply docs/internals/security.rst:208]: Yes, please revert.

File: docs/quickstart.rst [hunk 5]
```diff
@@ -400,6 +399,11 @@ For automated backups the passphrase can be specified using the
     A backup inside of the backup that is encrypted with that key/passphrase
     won't help you with that, of course.
 
+    In case you lose your repository and the security information, but have an
+    older copy of it to restore from, don't use that later for creating new
+    backups – you would run into security issues (reuse of nonce counter
+    values). It is better to initialize a new Borg repository.
+
```
Comment (ThomasWaldmann) [inline docs/quickstart.rst:406]: Point to docs added in last (merged) PR?

**Pull request closed without merging**